integration_tests/roborazzi/src/test/java/org/robolectric/integration/roborazzi/RoborazziCaptureTest.kt

```diff
@@ -0,0 +1,237 @@
+package org.robolectric.integration.roborazzi
+
+import android.app.Activity
+import android.app.AlertDialog
+import android.app.Application
+import android.content.ComponentName
+import android.graphics.BitmapFactory
+import android.graphics.Color
+import android.graphics.Rect
+import android.graphics.drawable.ColorDrawable
+import android.os.Build.VERSION_CODES.S
+import android.os.Bundle
+import android.view.Gravity
+import android.view.View
+import android.view.WindowManager
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.test.core.app.ActivityScenario
+import androidx.test.platform.app.InstrumentationRegistry
+import com.github.takahirom.roborazzi.DEFAULT_ROBORAZZI_OUTPUT_DIR_PATH
+import com.github.takahirom.roborazzi.ExperimentalRoborazziApi
+import com.github.takahirom.roborazzi.RoborazziOptions
+import com.github.takahirom.roborazzi.RoborazziTaskType
+import com.github.takahirom.roborazzi.captureScreenRoboImage
+import com.github.takahirom.roborazzi.roboOutputName
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.Shadows
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.GraphicsMode
+
+/**
+ * Integration Test for Roborazzi
+ *
+ * This test is not intended to obstruct the release of Robolectric.
+ * In the event that issues are detected which do not stem from Robolectric,
+ * the test can be temporarily disabled, and an issue can be reported on the Roborazzi repository.
+ */
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [S])
+@GraphicsMode(GraphicsMode.Mode.NATIVE)
+@OptIn(ExperimentalRoborazziApi::class)
+class RoborazziCaptureTest {
+  @Test
+  fun checkViewWithElevationRendering() {
+    hardwareRendererEnvironment {
+      registerActivityToPackageManager(RoborazziViewWithElevationTestActivity::class.java.name)
+      val activityScenario =
+        ActivityScenario.launch(RoborazziViewWithElevationTestActivity::class.java)
+
+      captureScreenWithRoborazzi()
+
+      // View the screenshots at build/output/roborazzi/*.png
```

Comment:
Are there golden images stored anywhere?

Reply:
Currently, we only check a single pixel of the image.
I've attempted to implement the saving of small screenshots in the Robolectric repository. However, there's a problem: we can't record the golden images on our local machines because the [images are non-deterministic](https://github.com/robolectric/robolectric/issues/8457). We could use GitHub Actions to commit them, similar to what's done in 'Now in Android'. But I believe Robolectric might be built on a Google server.